test/e2e/gitops_service_test.go

```diff
@@ -237,3 +245,34 @@ func tearDownArgoCD(t *testing.T) {
 	assertNoError(t, err)
 
 }
+
+func deployClusterScopedManifest(t *testing.T) {
+	framework.AddToFrameworkScheme(configv1.AddToScheme, &configv1.Image{})
+	f := framework.Global
+	path, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+	imageYAML := filepath.Join(path, "yamls", "image.yaml")
+	ocPath, err := exec.LookPath("oc")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	cmd := exec.Command(ocPath, "apply", "-f", imageYAML)
+	err = cmd.Run()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	time.Sleep(5 * time.Second)
+
+	existingImage := &configv1.Image{
+		ObjectMeta: v1.ObjectMeta{
+			Name: "cluster",
+		},
+	}
+
+	err = f.Client.Get(context.TODO(), types.NamespacedName{Name: existingImage.Name}, existingImage)
+	assertNoError(t, err)
+}
```

Comment:
Should we delete the context like we do for any other test.
	ctx := framework.NewContext(t)
	defer ctx.Cleanup()

Reply:
Done